docs/guides/compose.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Compose
+---
+
+The Tailcall CLI's `compose` command facilitates the merging of configuration files and their conversion between different formats using the `--format` option.
+
+It supports `gql` or `graphql`, `yml` or `yaml`, `json`. Default is `json`.
+
+## Converting Formats with `compose --format`
```

Comment:
avoid writing monospace in headings. Bad SEO and doesn't look that good when HTML is rendered.